microsoft_intune/help.md

```diff
@@ -261,6 +261,7 @@ _This plugin does not contain any troubleshooting information._
 
 # Version History
 
+* 1.2.2 - Update docs_url in plugin.spec.yaml
```

Comment:
@ekim-r7 if you check the other plugins, we have a standard message for this change

```suggestion
* 1.2.2 - Add `docs_url` to plugin spec with link to [plugin setup guide](https://docs.rapid7.com/insightconnect/microsoft-intune/)
```

e.g. 

```
$ grep docs_url */help.md
aws_securityhub/help.md:* 2.0.2 - Add `docs_url` to plugin spec with link to [plugin setup guide](https://insightconnect.help.rapid7.com/docs/aws-security-hub)
jira/help.md:* 6.0.3 - Add `docs_url` to plugin spec with link to [plugin setup guide](https://docs.rapid7.com/insightconnect/jira)
microsoft_atp/help.md:* 4.5.1 - Add `docs_url` to plugin spec with link to [plugin setup guide](https://docs.rapid7.com/insightconnect/microsoft-defender-ATP)
microsoft_teams/help.md:* 2.0.3 - Add `docs_url` to plugin spec with link to [plugin setup guide](https://insightconnect.help.rapid7.com/docs/microsoft-teams)
powershell/help.md:* 2.1.2 - Add `docs_url` to plugin spec with link to [plugin setup guide](https://insightconnect.help.rapid7.com/docs/mass-delete-with-powershell)
python_3_script/help.md:* 2.0.2 - Add `docs_url` to plugin spec with link to [plugin setup guide](https://insightconnect.help.rapid7.com/docs/python-2-or-3-script)
rest/help.md:* 3.0.3 - Add `docs_url` to plugin spec with link to [plugin setup guide](https://insightconnect.help.rapid7.com/docs/rest)
```